packages/examples/packages/webpack-plugin/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@metamask/webpack-plugin-example-snap",
+  "version": "0.34.1-flask.1",
+  "description": "MetaMask example snap demonstrating how to use the Webpack plugin to bundle a snap.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snaps.git"
+  },
+  "license": "ISC",
+  "sideEffects": false,
+  "main": "dist/bundle.js",
+  "files": [
+    "dist/",
+    "images/icon.svg",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "webpack",
+    "build:clean": "yarn clean && yarn build",
+    "clean": "rimraf \"dist/*\"",
+    "lint": "yarn lint:eslint && yarn lint:misc --check && yarn lint:changelog",
+    "lint:changelog": "yarn auto-changelog validate",
+    "lint:ci": "yarn lint",
+    "lint:eslint": "eslint . --cache --ext js,ts,jsx,tsx",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" \"!snap.manifest.json\" --ignore-path ../../../../.gitignore",
+    "test": "yarn test:e2e",
+    "test:e2e": "jest",
+    "start": "webpack watch"
```

Comment:
Do we not need `mm-snap serve`?

Reply:
Ideally, Webpack is able to do this without the need for `mm-snap`. I haven't looked so much into the `start` command yet. Will do that when setting up the site.